modules/ROOT/pages/dataweave-system-properties.adoc

```diff
@@ -25,59 +37,93 @@ By default, adds to coercion exceptions more information about data that fails t
 * Type: `Boolean`
 * Default: `true`
 
-//com.mulesoft.dw.date_minus_back_compatibility//
-|`com.mulesoft.dw.date_minus_back_compatibility`|
-Restores the xref:dataweave-cookbook-add-and-subtract-time.adoc[Add and Subtract Time] behaviors that changed in 2.3.0 when this property is set to `true`.
+
+//com.mulesoft.dw.cpulimit.watchdog//
+|`com.mulesoft.dw.cpulimit.watchdog`|
+Monitors and limits script execution time when set to `true`.
 
 * Type: `Boolean`
-* Default: `false`
+* Default: `true`
 
-Available for language levels: 2.3, 2.4
 
-Example:
+//com.mulesoft.dw.date_minus_back_compatibility//
+|`com.mulesoft.dw.date_minus_back_compatibility`|
+
+Restores the xref:dataweave-cookbook-add-and-subtract-time.adoc[Add and Subtract Time] behaviors that changed in 2.3.0 when this property is set to `true`.
```

Comment:
I'm not sure I understand this one. Is it changing the behavior back to what it was in 2.3.0 or to versions earlier than 2.3.0?

Reply:
In 2.3.0 the behavior changed. This property restores the behavior Add and Subtract Time used to have.

Reply:
Thanks for explaining.